R/spacy_initialize.R

```diff
@@ -72,14 +78,20 @@ spacy_initialize <- function(model = "en",
     # if(! lang %in% c('en', 'de')) {
     #     stop('value of lang option should be either "en" or "de"')
     # }
+    omit_entity <- !entity
     spacyr_pyassign("model", model)
+    spacyr_pyassign("omit_entity", omit_entity)
+    options("omit_entity" = omit_entity)
     spacyr_pyexec(pyfile = system.file("python", "initialize_spacyPython.py",
                                        package = 'spacyr'))
     # spacy_version <- system2("pip", "show spacy", stdout = TRUE, stderr = TRUE)
     # spacy_version <- grep("Version" ,spacy_version, value = TRUE)
     # 
     spacy_version <- spacyr_pyget("versions")$spacy
-
+    if(omit_entity == TRUE & substr(spacy_version, 1, 1) != "2"){
+        message("entity == FALSE is only available for spaCy version 2.0.0 or higher")
+        options("omit_entity" = FALSE)
```

Comment:
If we have multiple options for spacyr, I suggest calling them `spacyr_entity` and making the default `FALSE`. This means that 
a) all of the spacyr options are grouped in the options list, and   
b) the names and values correspond to the `spacy_initialize()` options.